zeek-config.in

```diff
@@ -100,7 +100,8 @@ while [ $# -ne 0 ]; do
             echo $build_type
             ;;
         --caf_root)
-            echo $caf_root
+            echo "The caf_root option is deprecated and will be removed in 5.1. The Broker API has been updated to no longer require access to CAF to build against Broker."
+            exit 1
```

Comment:
"Deprecated" suggests that usage won't break things for now, so I'd remove the exit: just ignore and continue.